tests/test_generate_plots.py

```diff
@@ -61,10 +61,10 @@ def test__generate_stacked_bar_plot_data(self):
             [
                 {
                     "2-3": 5.0,
-                    "4-5": 5.0,
-                    "6-10": 30.0,
-                    ">10": 21.66666666666667,
-                    "FO/NP": 33.33333333333333,
+                    "4-5": 10.0,
+                    "6-10": 20.0,
+                    ">10": 10.0,
+                    "FO/NP": 50.0,
```

Comment:
Is this going to be changed to 'Missing' or something more obvious at some stage?